src/daemon/sid.c

```diff
@@ -122,8 +104,6 @@ static void _become_daemon()
 
 	umask(SID_DEFAULT_UMASK);
 
-	if (kill(getppid(), SIGTERM) < 0)
```

Comment:
There's only one implication of removal of this child to parent signalling...

When daemonizing, there's some little initialization done in the child process (after `fork` and before `kill` in the `_become_daemon` function) and if anything failed during this time, the child could still signal the parent about the failure and the parent could then return that status to the calling process (e.g. a command line from which SID is executed). However, the initialization is only `setsid`, `chdir` and a few `dup2` calls, so the signalling code is probably not worth for this. It would make sense if we had more complex initialization, like including reinstating the internal daemon state from permanent storage. So ok, let's dump it...